### ISSUE TITLE
How to handle nodes with labeled information?

### ISSUE THREAD

**Issue by wzc118** [tkipf/gcn#82]
How can I handle the GCN  with node labeled information for a community detection problem? For example, each node has some attributes (categorical features), how can I combine them then feed into GCN?  Like word embedding for texts or CNN extraction for images.  Also use one-hot?
Also , Is it a measurable solution to generate new features combining raw labeled features with link features with GAE?

**Comment by tkipf** [comment]
Yes, you can choose any input feature vector that you like (assuming
features are scaled properly -- a good rule of thumb: if it works in an
MLP, it'll work in a GCN).

I don't understand your last question about GAEs, could you elaborate?

**Comment by wzc118** [comment]
Thxs, Now I enconter with some other questions, how to handle the data with time , which means at time1 nodeA and nodeB has a link, then at time2 ... Can the adj matrix to be a weighted matrix?
